### DISCUSSION TITLE
Support loading binaries to path

### DISCUSSION THREAD

**Discussion by kylo252** [marlonrichert/zsh-snap#74] [Ideas]
I was looking at https://github.com/bigH/git-fuzzy and their README describes how to install it with either `antibody` or `zplug`. 

>   ### Antibody
>   Update your `.zshrc` file with the following line:
>  ```
>  antibody bundle bigH/git-fuzzy path:bin kind:path
>  ```

>  ### zplug
>  ```
>  zplug "bigH/git-fuzzy", as:command, use:"bin/git-fuzzy"
>  ```

I don't think this is currently possible with `znap`, is it? If not, how hard would it be to implement such a feature?

**Comment by marlonrichert** [comment]
Yes, it's already possible with Znap. Just add it to your path like this:
```zsh
path=( ~[bigH/git-fuzzy]/bin $path )
```
Znap will automatically download it for you.

**Reply by marlonrichert** [reply]
I submitted a pull request to `git-fuzzy` to include this in their instructions: https://github.com/bigH/git-fuzzy/pull/36

**Reply by kylo252** [reply]
Sorry for the late reply and thanks for the nice tool as usual :)

Two questions: 

1- Wouldn't it make more sense to symlink the file to a single `znap-bin` directory instead of adding it directly to `$path`? I'm just thinking it's gonna get crowded pretty quickly if you add a few of these. 
2- If you agree with the first point, maybe consider adding a nice wrapper command? 

As a reference, I think `zplug` is using [symlinks](https://github.com/zplug/zplug/blob/master/base/core/load.zsh) as well but I can't say for certain due to my limited experience with zsh scripting.

**Reply by marlonrichert** [reply]
All right, it's now implemented: 4cd535e. Enjoy!

---

If you use my software on a regular basis, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
